jobs/wf1-2-createPatients.js

```diff
@@ -0,0 +1,102 @@
+//Define gender options and prepare newPatientUuid and identifiers
+fn(state => {
+  const genderOptions = {
+    male: 'M',
+    female: 'F',
+    unknown: 'U',
+    'Transgender female': 'O',
+    'Transgender male': 'O',
+    'Prefer not to answer': 'O',
+    'Gender variant - Non conforming': 'O',
+  };
+
+  const identifiers = [];
+  const newPatientUuid = [];
+
+  return { ...state, genderOptions, newPatientUuid, identifiers };
+});
+
+// Creating identifiers for each patient
+each('trackedEntityInstances[*]', state => {
+  return post(
+    'idgen/identifiersource/8549f706-7e85-4c1d-9424-217d50a2988b/identifier',
```

Comment:
I can't find any documentation for this API, but a couple of things to think about:

1) Can we not ask for multiple ids in one request?
2) The adaptor should add support for this

Reply:
@josephjclark i don't think it's possible to create multiple ids, in one request. Couple of read that i ran into 
- https://talk.openmrs.org/t/how-to-generate-patient-identifier-in-patient-registration/32487/11
- https://rest.openmrs.org/#create-remoteidentifiersource

For adding support, i totally agree but we don't have the right documentation for this.